truss/remote/baseten/remote.py

```diff
@@ -33,12 +33,30 @@
 from truss.util.path import is_ignored, load_trussignore_patterns
 from watchfiles import watch
 
+API_URL_MAPPING = {
+    "https://app.baseten.co": "https://api.baseten.co",
+    "https://app.staging.baseten.co": "https://api.staging.baseten.co",
+    "https://app.dev.baseten.co": "https://api.staging.baseten.co",
+    # For local development, this is how we map URLs
+    "http://localhost:8000": "http://api.localhost:8000",
+}
+
+# If a non-standard domain is used with the baseten remote, default to
+# using the production api routes
+DEFAULT_API_DOMAIN = "https://api.baseten.co"
```

Comment:
we really only hit this case in tests. I add this mapping so that we don't have to have a separate API domain specified in the ~/.trussrc.